src/main/scala/me/rexim/issuestant/Main.scala

```diff
@@ -48,6 +48,7 @@ object Main {
   }
 
   def main(args: Array[String]): Unit = {
+    @SuppressWarnings(Array("org.wartremover.warts.AsInstanceOf"))
```

Comment:
I was forced to check your commit comment to learn why do we need that.

I suggest to put a comment above every non-obvious warning suppression statement.